python-lib/dku_stress_test_center/stress_test_center.py

```diff
@@ -246,6 +246,11 @@ def build_results(self):
 
         df = self.model_accessor.get_original_test_df(sample_fraction=self._sampling_proportion,
                                                       random_state=self._random_state)
+
+        target_map = self.model_accessor.get_target_map()
+        target = self.model_accessor.get_target_variable()
+        df = df[df[target].isin(target_map.keys())]
```

Comment:
[Nitpick]
Can be written simply like this:
```suggestion
        df = df[df[target].isin(target_map)]
```

Reply:
Didn't know about that ! Thanks